core/templates/dev/head/components/forms/ObjectEditorDirective.js

```diff
@@ -25,7 +25,9 @@ oppia.directive('objectEditor', ['$compile', '$log', function($compile, $log) {
       initArgs: '=',
       isEditable: '@',
       objType: '@',
-      value: '='
+      value: '=',
+      validators: '&',
```

Comment:
getValidators: '&validators'

Reply:
The `validator` and `uiConfig` naming of variables is used in many places throughout the code. Hence I used it. It starts from `schema_based_editor_directive` as can be seen [here](https://github.com/oppia/oppia/blob/develop/core/templates/dev/head/components/forms/schema_editors/schema_based_editor_directive.html#L44). If I use different names here, naming will become inconsistent.

Reply:
That's fair. Thanks for checking and for pushing back on this. I agree with you!